### ISSUE TITLE
Rules Model is too complex

### ISSUE THREAD

**Issue by wcandillon** [28msec/nolap-report-editor#67]
https://codeclimate.com/github/28msec/nolap-report-editor/app/modules/rules-model.js

This will make futur refactoring fairly complex. One possibility would be to use a mustache template to generate the rule. Here's an example: https://github.com/wcandillon/swagger-js-codegen/blob/master/templates/method.mustache

**Comment by dknochen** [comment]
yes, I had the same idea. I created a user story for this: https://28.tpondemand.com/RestUI/TpView.aspx?acid=7A5AD34FC89B789A76703F489A96FB11#userstory/7199
